pom.xml

```
@@ -98,8 +98,7 @@ under the License.
     <dependency>
       <groupId>org.pac4j</groupId>
       <artifactId>pac4j-saml</artifactId>
-      <!-- versions 4.x.x require JDK 11 -->
-      <version>3.9.0</version>
+      <version>5.2.0</version>
```

Why not the latest version `5.4.6`?

```suggestion
      <version>5.4.6</version>
```